ir/visitor.h

```diff
@@ -134,6 +134,12 @@ class Visitor {
      * control flow graph.  Should update @this and leave the other unchanged.
      */
     virtual void flow_merge(Visitor &) { }
+    /** Support methods for non-local ControlFlow computations */
```

Comment:
do you mean inter-procedural?

Reply:
Can be, but can also be used for goto-like things within a function (eg, exit/return/break).  They're 'non local' in that they mean control flow from an IR node to another IR node that is not a child or parent of that node.